packages/core/tests/ledger.test.ts

```diff
@@ -57,11 +55,26 @@ describe('ledger', () => {
       expect.objectContaining({
         did: faberAgent.publicDid.did,
         verkey: verkey,
-        role: '101',
+        role: '0',
       })
     )
   })
 
+  test('register public DID on ledger', async () => {
+    if (!faberAgent.publicDid) {
+      throw new Error('Agent does not have public did.')
+    }
+
+    const targetDid = 'PNQm3CwyXbN5e39Rw3dXYx'
+    const targetVerkey = '~AHtGeRXtGjVfXALtXP9WiX'
+
+    const result = await faberAgent.ledger.registerPublicDid(targetDid, targetVerkey, 'alias', 'TRUST_ANCHOR')
+
+    await sleep(2000)
```

Comment:
Is this sleep necessary? Is the result object mutated after the value is returned?

Reply:
True! Not necessary, forgot to remove it when copy-pasting tests :-P